package.json

```diff
@@ -31,13 +31,13 @@
     "bootstrap-sass": "^3.3.5",
     "classnames": "^2.1.3",
     "event-source-polyfill": "^0.0.7",
-    "highlight.js": "^8.9.1",
+    "highlight.js": "^9.3.0",
     "lodash": "^4.11.2",
     "moment": "^2.10.6",
     "postal": "^2.0.0",
     "postal.diagnostics": "^0.7.4",
-    "react": "^0.13.3",
-    "react-highlight": "^0.5.1",
+    "react": "^15.0.2",
```

Comment:
It would be nice to get to at least `react` `14.x.x`, which then allows us to use `react-redux` (if and when we decide to start pulling in some of those concepts).